src/main/java/alns/Evaluator.java

```diff
@@ -0,0 +1,42 @@
+package alns;
+
+import data.Problem;
+import objects.Order;
+
+import java.util.List;
+
+public class Evaluator {
+
+    public static boolean evaluateLoad(List<List<Order>> orderSequences) {
+        for(int i = 0; i < orderSequences.size(); i++) {
+            double currentLoad = findTotalStartLoad(orderSequences.get(i));
+            if(currentLoad > Problem.vessels.get(i).getCapacity()) return false;
+            for(int j = 0; j < orderSequences.get(i).size(); j++) {
```

Comment:
Her kan enhanced for loop være cleanere:
`for (Order order : orderSequences.get(vesselNumber))`